BitwardenShared/UI/Vault/Vault/ImportLogins/ImportLoginsView.swift

```diff
@@ -35,20 +56,83 @@ struct ImportLoginsView: View {
         .padding(.top, 8)
         .frame(maxWidth: .infinity)
         .scrollView()
-        .navigationBar(title: Localizations.importLogins, titleDisplayMode: .inline)
-        .toolbar {
-            cancelToolbarItem {
-                store.send(.dismiss)
+    }
+
+    /// The step 1 page view.
+    @ViewBuilder
+    private func step1() -> some View {
+        stepView(step: 1, totalSteps: 3, title: Localizations.exportYourSavedLogins) {
+            NumberedListRow(title: Localizations.exportLoginsStep1)
+            NumberedListRow(title: Localizations.exportLoginsStep2)
+            NumberedListRow(
+                title: Localizations.exportLoginsStep3,
+                subtitle: Localizations.exportLoginsStep3Subtitle
+            )
+        }
+    }
+
+    /// Returns a scroll view for displaying the instructions for a step.
+    ///
+    /// - Parameters:
+    ///   - step: The step number of this page.
+    ///   - totalSteps: The total number of steps.
+    ///   - title: The title of the step.
+    ///   - list: A closure that returns the views to display in a numbered list.
+    ///
+    @ViewBuilder
+    private func stepView(
+        step: Int,
+        totalSteps: Int,
```

Comment:
🤔 Could this `totalSteps` be extracted to a constant at the file level? Maybe I'm misunderstanding this but I feel this value will be the same every time this func is called.

Reply:
Good point, changes are in latest PR.